src/backend/marsha/settings.py

```diff
@@ -405,6 +405,7 @@ class Base(Configuration):
             },
         }
     )
+    REMINDERS_SECRET_HASH = values.Value()
```

Comment:
I still don't think that the suffix `HASH` is needed. It's confusing. This setting is not a hash. Maybe a `SECRET_KEY` is more relevant.